### DISCUSSION TITLE
Enhanced transparency/visibility options

### DISCUSSION THREAD

**Discussion by spongbros** [OpenRCT2/OpenRCT2#15665] [Feature requests] [answered]
Currently, OpenRCT2 has See-Through Rides/Scenery and Invisible Supports/People options inherited from RCT2. These could be expanded so that rides and scenery could also be made invisible. See-Through and Invisible paths would also be a very welcome addition (#1874).

OpenTTD, by example, has a [Transparency Options](https://wiki.openttd.org/Transparency_options) toolbar that allows choosing between See-Through and Invisible for each type of object as well as fast toggling between the two views with the ability to lock the visibility of individual object types. (It should be noted that OpenTTD has no Underground/Inside View.)

**Comment by seifer7** [comment]
I'm working on this and #1519 ("See-through rides" doesn't affect all rides)

**Comment by MASHzj** [comment, marked as answer]
A Transparency Options window has now been implemented where you can do a lot of these things.
